### DISCUSSION TITLE
User claim incomplete error

### DISCUSSION THREAD

**Discussion by tiagogalvao7** [oidc-wp/openid-connect-generic#553] [General]
Good night, after several hours and several attempts to configure Keycloak and your plugin, this error still appears. I don't know what else to do, I followed your tutorial and your tips. Best regards!
![Screenshot_1](https://github.com/user-attachments/assets/c2ebaf25-891f-46ff-ab08-e6e25e04b3b8)

**Comment by timnolte** [comment]
What version of Keycloak are you using?

**Reply by tiagogalvao7** [reply]
hello, 24.0.2

**Reply by timnolte** [reply]
I'll have to confirm what version I have running and test. That error means that something is missing from the user claim being sent by Keycloak.

**Reply by tiagogalvao7** [reply]
Thank you! I'll be waiting!

**Reply by tiagogalvao7** [reply]
I tried again with Keycloak same version, but this time with Keycloak in dev mode, and it worked, but I need it to work with Keycloak in Production mode
